### PR TITLE
Adding radarx

### PR DESCRIPTION
**Pull request by syedhamidali**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by syedhamidali** [comment]
@conda-forge/help-python, ready for review!

File: recipes/radarx/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+{% set version = "0.1.3.dev0" %}
+{% set sha256 = "6551edac03645cf9cb76cd8b816cabd6dfa5c1eda49ed2eaba9f9e67adcc803c" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```
Comment (xhochy) [inline recipes/radarx/meta.yaml:11]: Please write it here directly and avoid unnecessary Jinja.

File: recipes/radarx/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+{% set name = "radarx" %}
+{% set version = "0.1.3.dev0" %}
```
Comment (xhochy) [inline recipes/radarx/meta.yaml:2]: Can you use a release version here, please.

File: recipes/radarx/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    - pip
+  run:
+    - python >=3.8
+    - xarray
+    - xarray-datatree
+    - xradar
+    - netcdf4
+    - cartopy >=0.20.0
+
+test:
```
Comment (xhochy) [inline recipes/radarx/meta.yaml:33]:
```suggestion
test:
  requires:
    - pip
  commands:
    - pip check
```

**Review by xhochy — CHANGES_REQUESTED** [review]

**Comment by syedhamidali** [comment]
@xhochy Thank you for your feedback. I have made all the suggested changes. Could you please review it again?

**Pull request merged**